src/test/java/com/wikia/webdriver/common/dataprovider/TrackingOptInDataProvider.java

```diff
@@ -46,6 +45,8 @@
       "https?://.*pubads\\.g\\.doubleclick\\.net/gampad/ads.*output=xml_vast.*cust_params=.*npa%3D1.*";
   private static final String ADS_VAST_NPA_SECOND_PARAMETER_PATTERN =
       "https?://.*pubads\\.g\\.doubleclick\\.net/gampad/ads.*output=xml_vast.*npa=1.*";
+  private static final String PETAMETRICS_PATTERN =
```

Comment:
Ho ho what is this?

Reply:
PETAMETRICS_PATTERN - request which is tracking for trending

Reply:
Nice, I didn't know that! Thanks for sharing :)